citing-us/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Citing OME
+---          
+        <div class="callout large primary" id="bg-image-licensing">
+            <div class="row column text-center">
+                <h1>Citing Us</h1>
+                <p class="hero-subheader small-12 medium-12">We are delighted to have you cite the work of the Open Microscopy Environment from your own publications.</p>
+            </div>
+        </div>
+        
+        <!-- begin citation--> 
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-12 medium-4 columns">
+                <h3>OME-XML &amp; OME-TIFF</h3>
+            </div>
+            <div class="small-12 medium-8 columns">
+                <p>The 2010 article outlining the importance of metadata and introducing the OME Compliant Specification:</p>
+                <p class="citation">Melissa Linkert, Curtis T. Rueden, Chris Allan, Jean-Marie Burel, Will Moore, Andrew Patterson, Brian Loranger, Josh Moore, Carlos Neves, Donald MacDonald, Aleksandra Tarkowska, Caitlin Sticco, Emma Hill, Mike Rossner, Kevin W. Eliceiri, and Jason R. Swedlow (2010) Metadata matters: access to image data in the real world. The Journal of Cell Biology, Vol. 189no. 5777-782 doi: 10.1083/jcb.201004104</p>
+                <p><a class="pmid" href="http://www.ncbi.nlm.nih.gov/pubmed/20513764" target="_blank">20513764</a></p>
+                <p>The original OME Data Model paper from 2005:</p>
+                <p class="citation">Goldberg, I., C. Allan, J.-M. Burel, D. Creager, A. Falconi, H. Hochheiser, J. Johnston, J. Mellen, P.K. Sorger, and J.R. Swedlow. (2005) The Open Microscopy Environment (OME) Data Model and XML File: Open Tools for Informatics and Quantitative Analysis in Biological Imaging. Genome Biol. 6:R47.</p>
+                <p><a class="pmid" href="http://www.ncbi.nlm.nih.gov/pubmed/15892875" target="_blank">15892875</a></p>
+            </div>            
+        </div>      
+        <!-- end citation-->
+        
+        <!-- begin citation--> 
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-12 medium-4 columns">
+                <h3>Bio-Formats</h3>
+            </div>
+            <div class="small-12 medium-8 columns">
+                <p class="citation">Melissa Linkert, Curtis T. Rueden, Chris Allan, Jean-Marie Burel, Will Moore, Andrew Patterson, Brian Loranger, Josh Moore, Carlos Neves, Donald MacDonald, Aleksandra Tarkowska, Caitlin Sticco, Emma Hill, Mike Rossner, Kevin W. Eliceiri, and Jason R. Swedlow (2010) Metadata matters: access to image data in the real world. The Journal of Cell Biology, Vol. 189no. 5777-782 doi: 10.1083/jcb.201004104</p>
```

Comment:
Having pasted the content without checking the older entries thoroughly, this should be '...Journal of Cell Biology 189(5), 777-782.